backend/alanda-springboot-mixin/pom.xml

```diff
@@ -280,25 +259,35 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
+        <version>${maven-surefire-plugin.version}</version>
         <configuration>
+          <includes>
+            <include>**/*Test.java</include>
```

Comment:
is the configuration update necessary? afaik this is part of the default config
same for **IT below